frontend/modules/recipe/components/RecipeScheme.js

```diff
@@ -120,20 +120,20 @@ class RecipeScheme extends React.Component {
   render() {
     const {formatMessage} = this.props.intl;
     const messages = defineMessages({
-      cooking_time: {
-        id: 'recipe.cooking_time',
-        description: 'Cooking time',
-        defaultMessage: 'Cooking time',
+      servings: {
+        id: 'recipe.servings',
+        description: 'Servings',
+        defaultMessage: 'Servings',
```

Comment:
Theres no need to swap these. It doesn't hurt, but they're just there for setting the translations.